src/Apps/Order2/Routes/Details/Components/Order2DetailsFulfillmentInfo.tsx

```diff
@@ -21,14 +21,14 @@ export const Order2DetailsFulfillmentInfo: React.FC<
   const isPickup = selectedFulfillmentOption?.type === "PICKUP"
 
   return (
-    <Box p={2} backgroundColor="mono0">
+    <Box p={4} backgroundColor="mono0">
```

Comment:
I think we want to keep both `2` and `4` in here
so we could only use 2 for mobile and 4 for desktop
so it would be something like this:
```tsx
    <Box p={[2, 4]} backgroundColor="mono0">
```

for more info, check out our [Palette doc](https://palette.artsy.net/guides/responsive/) about responsive props